src/librustdoc/html/static/rustdoc.css

```diff
@@ -1697,6 +1701,9 @@ details.rustdoc-toggle > summary.hideme {
 	cursor: pointer;
 }
 
+details.rustdoc-toggle > summary, details.undocumented > summary {
+	list-style: none;
```

Comment:
What necessitated this? Also, assuming we go forward with this I would rather add the `rustdoc-toggle` class when we emit `undocumented` toggles, instead of having two separate CSS selectors for this.

Reply:
It's because of chromium. The `<details>` arrow is still buggy on it so it needs extra rules.

Reply:
chromium as opposed to chrome? or both chrome and chromium?

And what's the display bug that happens without this?

Reply:
No, chromium specifically, not chrome and I assume not opera/safari/edge.

As for the bug, it shows the default `<details>` "arrow" alongside our summary text. It doesn't break the display, just looks weird.